halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/GateBoot154ProfileFactory.java

```diff
@@ -41,6 +41,13 @@ protected GateConfig getGateConfig(ServiceSettings gate, Security security) {
       config.x509 = new X509Config(security);
     }
 
+    if (security.getAuthn().isEnabled()) {
+      config.management = new Management();
+      config.spring = new SpringConfig();
+      config.endpoints = new GateConfig.Endpoints();
+      gate.setHealthPort(config.management.port);
```

Comment:
We will likely have to make a GateBoot19Profile factory since this change will only affect spinnaker releases after that point, right?